experimental/docker-compose/README.md

```diff
@@ -37,7 +37,45 @@ Under Kitematic, click on the application 'deck' -> settings -> ports. Click on
 
 <img src="https://cloud.githubusercontent.com/assets/74310/11158618/4bba7122-8a0e-11e5-83b6-8ff2297562b2.png"/>
 
-Alternatively, you can just enter ```DOCKER_IP=`docker-machine ip default` && open http://$DOCKER_IP:9000```
+Alternatively, you can just enter ```open http://`docker-machine ip default`:9000```
+
+## Using locally built images
+
```

Comment:
you only need docker ip when you are running but not when you are calling it to build an image.